life_expectancy/cleaning.py

```diff
@@ -3,7 +3,6 @@
 """
 
 from pathlib import Path
-from typing import List
 import argparse
 import logging
 import pandas as pd
```

Comment:
**suggestion** There a linter called `isort` that sorts imports alphabetically, and automatically separated into sections and by type. You might like it 😉

The benefit is that you libraries are always located in similar positions relative to one another in all your modules

Reply:
already imported it and run locally. should I add this to the CI (not sure that it makes sense since the runner is not going to change the cleaning.py file?), or to the pre-commit as you showed?